Tests/SignalRClientTests/Fakes.swift

```diff
@@ -41,35 +41,44 @@ class TestConnectionDelegate: ConnectionDelegate {
 }
 
 class TestHttpClient: HttpClientProtocol {
-    private var getHandler: ((URL) -> (HttpResponse?, Error?))?
-    private var postHandler: ((URL) -> (HttpResponse?, Error?))?
+    
+    typealias RequestHandler = (URL) -> (HttpResponse?, Error?)
+    
+    private var getHandler: RequestHandler?
+    private var postHandler: RequestHandler?
+    private var deleteHandler: RequestHandler?
 
-    init(getHandler: ((URL) -> (HttpResponse?, Error?))?, postHandler: ((URL) -> (HttpResponse?, Error?))?) {
+    init(getHandler: RequestHandler?, postHandler: RequestHandler?, deleteHandler: RequestHandler?) {
         self.getHandler = getHandler
         self.postHandler = postHandler
+        self.deleteHandler = deleteHandler
     }
 
     convenience init(getHandler: ((URL) -> (HttpResponse?, Error?))?) {
```

Comment:
It could do, yes. Looking at it again, these convenience initialisers can be removed and replaced with a default value of nil for each of the parameters of the main init, which I think is cleaner.